RELEASE/scripts/autoscend/auto_powerlevel.ash

```diff
@@ -52,6 +52,26 @@ boolean LX_attemptPowerLevel()
 		set_property("auto_powerLevelAdvCount", 0);
 		return true;		//restart the main loop to give those quests a chance to run now that the softblock is released.
 	}
+
+	if(get_property("screechDelay").to_boolean())
+	{
+		auto_log_warning("Patriotic Eagle's screech banished something we need and we can't adventure anywhere else");
+		if(get_property("screechCombats").to_int() > 0)
+		{
+			if(LX_getDigitalKey() || LX_getStarKey())
+			{
+				//If we still need the Digital Key or Star Key burn turns to get them
+				return true;
```

Comment:
I'd propose don't return anything here and just have a comment. This will allow proposed condition below to adv in noob cave to clear banish without going through the task list again-

Reply:
I think a better way to do this is a while screechCombats > 0. That way it will just keep looping here until we can screech